source/_docs/configuration/devices.markdown

```diff
@@ -37,8 +37,6 @@ sensor:
 
 switch:
   - platform: vera
-  - platform: tplink
-    host: IP_ADDRESS
```

Comment:
Any ideas for another example in place of tplink? Same applies to the change below.